powerline/lemonbar.py

```diff
@@ -0,0 +1,23 @@
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+
+from powerline import Powerline
+from powerline.lib.dict import mergedicts
+
+
+class LemonbarPowerline(Powerline):
```

Comment:
Missing `get_encoding` method. Not sure whether or not it is intentional, but without this method locale encoding will be used.